docker-compose-test.yml

```diff
@@ -4,7 +4,7 @@ services:
   wiremock:
     image: wiremock/wiremock
     networks:
-    - hmpps_int
```

Comment:
not sure why this is showing as a change, I made an accidental change to this file which I reverted but it's still showing a difference